examples/operator/templates/deployment.yaml

```diff
@@ -110,4 +110,13 @@ spec:
         name: manager-config
       - name: secret-volume
         secret:
-          secretName: {{ include "operator.fullname" . }}-secret-ca
\ No newline at end of file
+          secretName: {{ include "operator.fullname" . }}-secret-ca
+{{- if .Values.topologySpreadConstraints }}
+      topologySpreadConstraints: {{- include "tplvalues.render" (dict "value" .Values.topologySpreadConstraints "context" $) | nindent 8 }}
+{{- end }}
+{{- if .Values.nodeSelector }}
+      nodeSelector: {{- include "tplvalues.render" ( dict "value" .Values.nodeSelector "context" $) | nindent 8 }}
```

Comment:
Please generate samples with `generate-defaults=false`.  `nodeSelector` and `tolerations` are not presented in source manifest and should not be in the template by default.